.github/workflows/npm-download-stats.yml

```diff
@@ -0,0 +1,51 @@
+name: NPM Download Statistics
+
+on:
+  schedule:
+    # Run every Sunday at 00:00 UTC
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+permissions:
+  contents: write
```

Comment:
Nitpick: If I'm not mistaken, we usually set permissions on the job level. 🤔 

I.e.: 
```yml
permissions: {}
```
instead of this.
And the following after L14:
```yml
permissions:
  contents: write
```